src/java.base/share/classes/java/io/File.java

```diff
@@ -162,7 +165,8 @@
 "EnsuresNonNullIf(expression={\"list()\",\"list(FilenameFilter)\",\"listFiles()\",\"listFiles(FilenameFilter)\",\"listFiles(FileFilter)\"}, result=true)\""
 })
 @AnnotatedFor({"index", "initialization", "interning", "lock", "nullness"})
-public class File
+@DefaultQualifierForUse(Readonly.class)
```

Comment:
Should all files have this?

Reply:
I am still thinking. At least for most usage in `@RDM` class, this make sense because we need to make the most method can be used for both   `@Immutable` and `@Mutable` class like getters. Some methods can only be used by `@Mutable` receiver like `add(...)`.

I am wondering should we default the defaultqualifierforuse to RDM for RDM class?